app/client/api/authApi.js

```diff
@@ -157,14 +156,13 @@ export const ResetPasswordAPI = (payload) => {
 };
 
 export const authenticatePusher = () => {
-  console.log('authenticating pusher');
   startConfiguration({
     auth: {
       headers: {
         'Authorization': getToken()
       }
     },
-    authEndpoint: generateFormattedURL('/pusher/auth')
+    authEndpoint: '/api/v1/pusher/auth'
```

Comment:
Login bug here... was failing as generateFormattedURL needs to access login data